readme.md

```diff
@@ -260,6 +261,15 @@ To make a new Provider work, you need to extend base class
 `lobbyboy.provider.BaseProvider``, implement 2 methods:
 
 ```python
+    def is_available(self) -> bool:
+        """
+        Override this method to check for requirements of this provider
+
+        Returns:
+            bool: True if this provider is available, False to disable it
+        """
+        return True
+
```

Comment:
Maybe we can add a new part explanation(optional overrider) to readme, since only `create_server` and `destroy_server` are 2 `abstractmethod`, and others such as `is_available` and `ssh_server_command` are optional.